### PR TITLE
chore: add statsig release please configurations

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
## This PR

statsig release please configs were missing in the original PR

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -8,4 +8,5 @@
   "providers/unleash": "0.0.4-alpha",
   "providers/flipt": "0.0.2",
   "providers/configcat": "0.0.3"
```
Comment (liran2000) [inline .release-please-manifest.json]: add comma at the end

Comment (Kavindu-Dodan) [reply .release-please-manifest.json]: thanks & a good one :) 

updated and should be good to go

**Review by liran2000 — APPROVED** [review]

**Comment by toddbaert** [comment]
I think you'll need a matching `version.txt`. Otherwise I think it's good.

**Review by toddbaert — APPROVED** [review]

**Comment by Kavindu-Dodan** [comment]
> I think you'll need a matching `version.txt`. Otherwise I think it's good.

This is already there - https://github.com/open-feature/java-sdk-contrib/blob/main/providers/statsig/version.txt :)

**Pull request merged**